server.go

```diff
@@ -318,6 +297,53 @@ func (a *App) reply(w http.ResponseWriter, r *http.Request) {
 	}
 	successReply(w)
 }
+
+func (a *App) run(w http.ResponseWriter, r *http.Request) {
+	var msg Message
+	if err := json.NewDecoder(r.Body).Decode(&msg); err != nil {
+		errorReply(w, http.StatusBadRequest, "couldn't parse json")
+		return
+	}
+
+	msg.TwinSrc = a.twin
+	msg.Retqueue = uuid.New().String()
+
+	// if len(msg.TwinDst) > 1 || msg.TwinDst[0] != a.twin {
+	// 	errorReply(w, http.StatusBadRequest, "couldn't parse json")
+	// 	return
+	// }
+
+	err := a.backend.PushToLocal(r.Context(), msg)
+	fmt.Println(err)
```

Comment:
`Log.Error().err(err).Msg("something bad happened")` (we use log for everything to easily control what to be printed by the --debug flag)